specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -502,6 +502,64 @@
         }
       }
     },
+    "AcsEmailDeliveryReportReceivedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.EmailDeliveryReportReceived event.",
+      "type": "object",
+      "properties": {
+        "sender": {
+          "description": "The Sender Email Address",
+          "type": "string"
+        },
+        "recipient": {
+          "description": "The recipient Email Address",
+          "type": "string"
+        },
+        "messageId": {
+          "description": "The Id of the email been sent",
+          "type": "string"
+        },
+        "status": {
+          "description": "The status of the email",
+          "type": "string"
+        },
+        "deliveryAttemptTimeStamp": {
+          "description": "The time at which the email delivery report received timestamp",
+          "format": "date-time",
+          "type": "string"
+        }
+      }
+    },
+    "AcsEmailEngagementTrackingReportReceivedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.AcsEmailEngagementTrackingReportReceived event.",
+      "type": "object",
+      "properties": {
+        "sender": {
+          "description": "The Sender Email Address",
+          "type": "string"
+        },
+        "messageId": {
+          "description": "The Id of the email been sent",
+          "type": "string"
+        },
+        "userActionTimeStamp": {
+          "description": "The time at which the user interacted with the email",
+          "format": "date-time",
+          "type": "string"
+        },
+        "engagementContext": {
```

Comment:
Is there a set of known statuses that we expect or is this free form?

Reply:
there is no set of know status